stats/src/main/java/com/facebook/stats/mx/StatsReader.java

```diff
@@ -36,6 +36,8 @@
   public MultiWindowDistribution getDistribution(String key);
   public String getAttribute(StatType key);
   public String getAttribute(String key);
+  public Callable<Long> getDynamicCounter(StatType key);
+  public Callable<Long> getDynamicCounter(String key);
```

Comment:
I don't have any need for the Callable<> directly, but the other getFooCounter things returned the underlying object. Would you prefer it just return the Long?